src/frontend/src/common/components/library/data_subview/components/DeleteSamplesConfirmationModal/index.tsx

```diff
@@ -23,13 +24,22 @@ const DeleteSamplesConfirmationModal = ({
   const [shouldShowSuccessNotification, setShouldShowSuccessNotification] =
     useState<boolean>(false);
   const [numDeletedSamples, setNumDeletedSamples] = useState<number>(0);
+  const [isDeleteDisabled, setIsDeleteDisabled] = useState<boolean>(false);
   const { data: userInfo } = useUserInfo();
   const { group: userGroup } = userInfo ?? {};
 
   const samplesToDelete = checkedSamples
     .filter((sample) => sample.submittingGroup?.name === userGroup?.name)
     .map((sample) => sample.id);
 
+  useEffect(() => {
```

Comment:
does this need to be state? or can you say `isDeleteDisabled={isEmpty(samplesToDelete)}` on line 118?

Reply:
updated